src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -241,6 +241,12 @@
 WINDOWS_CDB_STACK_OVERFLOW_REGEX = re.compile(
     r'.*ExceptionCode: .*\(Stack overflow\).*')
 
+# Golang specific stack signatures.
+GOLANG_PANIC_REGEX = re.compile(r'^panic: (.*)')
```

Comment:
I suppose there isn't much we can do, but looking for the word "panic" might have some False positives.